### ISSUE TITLE
Route jobs away from `fasterq-dump` ?

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#873]
### Context

Related to https://github.com/AlexsLemonade/refinebio/pull/872, from that PR:

> Lots of our Salmon processors were getting stuck, but their containers weren't dying. This seemed to be occuring on the `fasterq-dump` step.

Some related issues for SRA tools: https://github.com/ncbi/sra-tools/issues/159, https://github.com/ncbi/sra-tools/issues/161

### Problem or idea

These hanging jobs will get killed as a result of #872, but will they run and hang again?

### Solution or next step

Some thoughts on what we might do next:
- Detect situations where `fasterq-dump` isn't working
- For these cases: use a different tool like `fastq-dump`, [`parallel-fastq-dump`](https://github.com/rvalieris/parallel-fastq-dump) (see: 
https://github.com/ncbi/sra-tools/issues/161#issuecomment-434729952) OR get these reads from European Nucleotide Archive 
- We could strip trailing Ns if that is the issue (https://github.com/ncbi/sra-tools/issues/159)

Tagging @Miserlou and @kurtwheeler to start the discussion

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by Miserlou** [comment]
I think I'd prefer to try to get them from ENA since the SRA file format was a stupid idea to begin with. We could also try updating the sra-tools version and trying again, I'm not sure if this bug is fixed already or not.

**Comment by jaclyn-taroni** [comment]
@Miserlou about what % of Salmon jobs hang? Is it a low enough number such that if we fall back to ENA, we won't have to be concerned about knocking things over?

**Comment by Miserlou** [comment]
Yes. This issue isn't actually our top priority right now, the corrupt (?) Salmon index issue is. Compared to that, it's only a small percentage of jobs that are getting stuck here.

**Comment by jaclyn-taroni** [comment]
May be related: https://github.com/ncbi/sra-tools/issues/168

**Comment by kurtwheeler** [comment]
I'm going to record an excerpt from a slack conversation between @Miserlou and I this morning that is relevant to this issue:

```
Rich [11:02 AM]
That would make sense, I don’t know what the under the hood differences are
We could also try retrying those samples from ena rather than ncbi

kurt-al [11:05 AM]
you mean like just the ones that fail?
or like try switching back to ENA and see if they can pipe us data faster?

Rich [11:06 AM]
ena will fall over if we do everything all at once from them
but maybe they won’t if we only do the ones that fail from ncbi?

kurt-al [11:07 AM]
I would think that it's not that something is wrong with those samples or those files, but rather we were running too many fasterqdumps at the same time so the I/O was a bottleneck that prevented fasterqdump from completing before the timeout (edited) 


Rich [11:08 AM]
yeah that’s true
```

Basically if there was any way to route jobs away from fasterq-dump that would be an incredible improvement because fasterqdump is a show-stopping problem right now. However we believe the issue is caused by the I/O bottleneck of writing all these files not from corrupt data, so just using another tool probably won't help and switching to ENA won't help because it can't serve us the data fast enough to relieve the I/O intensity of how many fasterq-dump processes we need to run.

**Comment by Miserlou** [comment]
The shorter term goal is to run fqd pointing to a tmpfs in ram so that we don't have the io problem, then to see if we can run with -e 64, which could drop 1 hours and io hell into about 2 minutes of isolate ram funtimes

**Comment by cgreene** [comment]
Moving to current sprint because this would be closed by #1106
